meta/runtime.yml

```
@@ -116,6 +116,10 @@ plugin_routing:
       deprecation:
         removal_version: 13.0.0
         warning_text: Project Atomic was sunset by the end of 2019.
+    bearychat:
+      tombstone:
+        removal_version: 11.2.0
```

Let's stick to the process and remove it from 12.0.0:
```suggestion
        removal_version: 12.0.0
```
We also need another PR deprecating it in 11.2.0.